src/components/foodCard/FoodCardSection.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import FoodCard from "./FoodCard";
+import { useQuery } from "styles/breakpoints";
+import { GridWrapper, FlexWrapper } from "components";
+import BlackButton from "components/buttons/BlackButton";
+
+const MapFoodSection = [
+  {
+    restaurantCardTitle: "Big and Juicy Wagyu Beef Cheeseburger",
+    image: "burger",
+    type: "Snack",
+    display: "show",
+    id: 1,
+  },
+  {
+    restaurantCardTitle: "Fresh Lime Roasted Salmon with Ginger Sauce",
+    image: "salmon",
+    display: "show",
+    type: "Fish",
+    id: 2,
+  },
+  {
+    restaurantCardTitle: "Strawberry Oatmeal Pancake with Honey Syrup",
+    image: "pancakes",
+    display: "show",
+    type: "Breakfast",
+    id: 3,
+  },
+  {
+    restaurantCardTitle: "Fresh and Healthy Mixed Mayonnaise Salad",
+    image: "healthy",
+    display: "show",
+    type: "Healthy",
+    id: 4,
+  },
+  {
+    restaurantCardTitle: "Chicken Meatballs with Cream Cheese",
+    image: "meatballs",
+    display: "",
+    type: "Meat",
+    id: 5,
+  },
+  {
+    restaurantCardTitle: "null",
+    image: "null",
+    display: "",
+    id: 99,
+  },
+
+  {
+    restaurantCardTitle: "Fruity Pancake with Orange & Blueberry",
+    image: "fruity",
+    display: "",
+    type: "Sweet",
+    id: 5,
+  },
+
+  {
+    restaurantCardTitle: "The Best Easy One Pot Chicken and Rice",
+    image: "chickenpot",
+    display: "",
+    type: "Snack",
+    id: 6,
+  },
+  {
+    restaurantCardTitle: "The Creamiest Creamy Chicken and Bacon Pasta",
+    image: "pasta",
+    display: "",
+    type: "Noodles",
+    id: 7,
+  },
+];
+
+const FoodCardSection: React.FC = () => {
+  const { isMobile, isTablet } = useQuery();
+
+  const [show, setShow] = useState<boolean>(false);
+
+  return (
+    <GridWrapper
+      gridTemplateColumns={isMobile ? "1fr" : isTablet ? "1fr" : "1fr 1fr 1fr"}
```

Comment:
too many conditionals. Enough only isTablet, and the style will be addapted on mobile breakpoint as well. by the way, why don't you use the object for breakpoints handling? 
also, more than one conditional in ternary operator is really bad practice. Most of our projects have eslint rule that throws an error if your'e using more than one conditional